cmd/cloudFoundryDeploy.go

```diff
@@ -226,46 +218,22 @@ func handleMTADeployment(config *cloudFoundryDeployOptions, command command.Exec
 }
 
 type deployConfig struct {
-	DeployCommand   string
-	DeployOptions   []string
-	AppName         string
-	ManifestFile    string
-	SmokeTestScript []string
+	DeployCommand string
+	DeployOptions []string
+	AppName       string
+	ManifestFile  string
 }
 
 func handleCFNativeDeployment(config *cloudFoundryDeployOptions, command command.ExecRunner) error {
 
-	deployType, err := checkAndUpdateDeployTypeForNotSupportedManifest(config)
-
-	if err != nil {
-		return err
-	}
-
 	var deployCommand string
-	var smokeTestScript []string
 	var deployOptions []string
 
 	// deploy command will be provided by the prepare functions below
 
-	if deployType == "blue-green" {
-		log.Entry().Warn("[WARN] Blue-green deployment type is deprecated for cf native builds " +
-			"and will be completely removed by 15.06.2024" +
-			"Instead set parameter `cfNativeDeployParameters: '--strategy rolling'`. " +
-			"Please refer to the Cloud Foundry documentation for further information: " +
-			"https://docs.cloudfoundry.org/devguide/deploy-apps/rolling-deploy.html." +
-			"Or alternatively, switch to mta build tool. Please refer to mta build tool" +
-			"documentation for further information: https://sap.github.io/cloud-mta-build-tool/configuration/.")
```

Comment:
We should have a similar warning message if somebody still comes along with deployment type blue-green, to tell them: This option has been removed now, do the following ...
The question then becomes: Should we switch to standard deployment, or should we fail. I would say the latter, as it would not be nice to overwrite an existing application with downtime when the user was expecting zero downtime.

Reply:
... and can we add a unit test to check this message getting printed?

Reply:
Since, we are cleaning up the docu itself, there is no option of choosing  deploytype as blue-green. Do you think it still makes sense?

Reply:
> ... and can we add a unit test to check this message getting printed?

Yes, added now!